database/sql/src/main/scala/cromwell/database/slick/tables/WorkflowMetadataSummaryEntryComponent.scala

```diff
@@ -55,7 +55,9 @@ trait WorkflowMetadataSummaryEntryComponent {
   )
 
   def filterWorkflowMetadataSummaryEntries(workflowStatuses: Set[String], workflowNames: Set[String],
-                                           workflowExecutionUuids: Set[String], startTimestampOption: Option[Timestamp],
+                                           workflowExecutionUuids: Set[String],
+                                           labelKeyLabelValues: Set[(String,String)],
+                                           startTimestampOption: Option[Timestamp],
```

Comment:
ToL: this seems like something you could declare separately and combine separately, to keep the pieces small and digestible.  I'm working to have something concrete to suggest but this is what my brain is saying.